src/praisonai/tests/unit/test_graph_memory.py

```
@@ -5,38 +5,39 @@
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..', 'praisonai-agents')))
 
 def test_memory_import():
     """Test that Memory class can be imported and initialized"""
     try:
         from praisonaiagents.memory import Memory
         print("✅ Memory class imported successfully")
-        return True
     except ImportError as e:
         print(f"❌ Failed to import Memory: {e}")
-        return False
+        assert False, f"Failed to import Memory: {e}"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)